sdk/core/AzureCore/Source/DataStructures/ResumableOperation.swift

```diff
@@ -50,6 +50,24 @@ public enum TransferState: Int16 {
             return "Deleted"
         }
     }
+
+    public var pauseable: Bool {
+        switch self {
+        case .unknown, .pending, .inProgress, .failed:
+            return true
```

Comment:
We talked about this after standup today, failed is considered resumable because that actually has "retry" semantics rather than resume. I'm still not clear whether it's correct that a failed transfer is also "pausable" though.

Reply:
Good point.  The intent was to intercept any "lost connection" failures to immediately pause and pre-empt the failed state. However, that isn't terribly reliable, so I had to make the failed state resumable. I agree--it shouldn't be both. I will make it simply resumable.

Reply:
This has been fixed.